pms-115-120/utility/src/main/java/io/mosip/pms/ida/dao/PartnerPolicyRepository.java

```diff
@@ -43,4 +43,10 @@ public interface PartnerPolicyRepository extends JpaRepository<PartnerPolicy, St
 
 	@Query(value = "select * from partner_policy pp where pp.part_id=?1 AND pp.policy_id=?2 AND pp.label=?3",nativeQuery = true)
 	public PartnerPolicy findByPartnerIdPolicyIdAndLabel(String partnerId, String policyId, String label);
+	
+	@Query(value = "select * from partner_policy pp where pp.cr_dtimes>?1 and pp.cr_dtimes<?2  and ( pp.is_deleted = false or pp.is_deleted is null)", nativeQuery = true)
+	public List<PartnerPolicy> findByCreatedDtimeGreaterThanAndIsDeletedFalseOrIsDeletedIsNull(LocalDateTime crDtimes, LocalDateTime presentDtimes);
```

Comment:
If some partner was available before and it is deleted now, don't we inform this to IDA, through websub ? This questions goes for other tables also.

Reply:
we don't have api's to delete sir. Any entry in pms can only be made inactive.

Reply:
Do you mean that is_deleted is never true, even when the partner is made inactive ?

Reply:
yes sir, only the value of the is_Active column will change.